azure-pipelines.yml

```diff
@@ -169,7 +169,7 @@ stages:
             XUnitPublishTargetFramework: net8.0
             IncludeDotNetCli: true
             DotNetCliPackageType: sdk
-            DotNetCliVersion: 8.0.100-rc.1.23455.8
+            DotNetCliVersion: 8.0.100-rtm.23506.1
```

Comment:
Does this need to be hardcoded? Can't we let it use the version from the repo's global.json?